ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml

```diff
@@ -158,3 +158,42 @@
   when:
     - ansible_architecture == "x86_64"
   tags: expat
+
+########
+# Java #
+########
+- name: Check if x86_64 java8 directory exists
```

Comment:
Instead of checking for both amd64 and x86-64 packages for  java 7 & 8 , couldn't you just check for if a package is available (repeat for java 7 and 8)?
Something like this:
```
- name: Check if java8 directory exists
   shell:  ls -d /usr/lib/jvm/java-1.8.0-openjdk.* >/dev/null 2>&1
   ignore_errors: yes
   register: java8
   tags: java
```
if `java8.rc == 0` then create symlink.